src/python/zquantum/core/cost_function.py

```diff
@@ -125,6 +126,7 @@ def __init__(
         n_samples: Optional[int] = None,
         epsilon: Optional[float] = None,
         delta: Optional[float] = None,
+        fixed_parameters: Optional[np.ndarray] = None,
```

Comment:
What are these used for? There should be a docstring for them/I don't understand how they're being used

Reply:
Fixed in [fa9842](https://github.com/zapatacomputing/z-quantum-core/pull/42/commits/fa9842b9d7331372acda6233df478ee8bbb15be4).